tardis/tests/integration_tests/report.py

```diff
@@ -67,14 +70,16 @@ def __init__(self, dokuwiki_details):
         else:
             self.dokuwiki_url = dokuwiki_details["url"]
 
+    def pytest_sessionstart(self, session):
```

Comment:
@yeganer `suite_start_time` is time zone unaware hence I cannot directly convert it to UTC. I won't have any idea how much offset is to be added / subtracted. Hardcoding +0200 according to CEST may not be a good idea either, I am in +0530 while debugging.